pkg/apis/beat/v1beta1/beat_types.go

```diff
@@ -37,6 +37,12 @@ type BeatSpec struct {
 	// +kubebuilder:validation:Optional
 	Config *commonv1.Config `json:"config,omitempty"`
 
+	// SecureSettings is a list of references to Kubernetes Secrets containing sensitive configuration options for the Beat.
+	// Secrets data can be then referenced in the Beat config using Secret keys or as specified in `Entries` field of
```

Comment:
```suggestion
	// Secrets data can be then referenced in the Beat config using the Secret's keys or as specified in `Entries` field of
```

I don't want sound super nit picky but this sentence still sounds like something is missing in front of Secret?